.github/workflows/ci-build.yml

```diff
@@ -10,9 +10,37 @@ on:
       - master
 
 jobs:
+  test-with-coverage:
+    name: Test with Coverage
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.14.x
+    
+    - name: Checkout code
+      uses: actions/checkout@v2
+    
+    - name: Generate Assets
+      run: |
+        make generate-assets
+ 
+    - name: Test
+      run: |
+        make test-with-coverage
+
+    - name: Upload code coverage
+      uses: codecov/codecov-action@v1
+      with:
+        files: ./build/_output/coverage/coverage.txt
+        flags: unittests # optional
+        fail_ci_if_error: true # optional (default = false)
+        verbose: true # optional (default = false)
+    
   golangci:
     name: GolangCI Lint
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
this should be the same https://github.com/actions/virtual-environments#available-environments

Reply:
yeah, but I wanted to make sure we stick to an LTS version even if `latest` changes.

Reply:
but do they use something else than LTS? It doesn't seem so...